applications/PfemBaseApplication/custom_utilities/mesh_data_transfer_utilities.cpp

```diff
@@ -1382,6 +1382,7 @@ namespace Kratos
 		//ELEMENT TRANSFER CHECK//
 	      }
 
+	    temporal_elements.clear();
```

Comment:
Is it needed anymore?  temporal_elements are destroyed at the end of the method.